benchmark/bootstrap_panoptes.rb

```diff
@@ -33,45 +33,53 @@ def create_db_schema
     @pg.exec <<-SQL
```

Comment:
I should have addressed this by the VOLUME mount in postgres docker-compose def.